hack/fossa.sh

```diff
@@ -2,6 +2,6 @@
 
 set -euo pipefail
 
-curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | bash
+curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash
```

Comment:
So there's an `install.sh` and `install-latest.sh`? I'm not sure how the versions mapping work here TBH.
Intuitively I would have thought that `latest` means something less stable. Is this the case?

Reply:
It is actually advised by FOSSA to do this: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/8774/pull-kubevirt-fossa/1618566997707067392#1:build-log.txt%3A62

So I figure this should be safe to do.